modules/end-user-guide/pages/adding-a-vscode-extension.adoc

```diff
@@ -0,0 +1,163 @@
+:navtitle: Adding a Visual Studio Code extension to a workspace
+:keywords: Visual Studio Code extension, user-guide
+
+[id="adding-visual-studio-code-extension_{context}"]
+= Adding a Visual Studio Code extension to a workspace
+
+Devfiles in the devfile v2 format don't contain any IDE-specific plug-ins or Visual Studio Code extensions. Instead, the plug-ins and extensions are described in a specific meta-folder.
+
+[id="plug-in-registries"]
+== {prod-short} plug-in registries overview
+
+{prod-short} has a default plug-in and extensions registry installed in every {prod-short} instance. The Che-Theia IDE searches and installs the plug-ins and extensions from that location.
```

Comment:
```suggestion
Every {prod-short} instance has a registry of default plug-ins and extensions. The Che-Theia IDE gets information about these plug-ins and extensions from the registry and installs them.
```
Are the plug-ins and extensions actually _in_ the registry, or does the registry point to their locations? This suggestion supposes the latter.

Reply:
point to the location in Che (but can be sometimes stored in the registry)